Aquality.Selenium/src/Aquality.Selenium/Configurations/WebDriverSettings/DriverSettings.cs

```diff
@@ -90,6 +91,24 @@ public virtual string DownloadDir
             }
         }
 
+        protected IReadOnlyList<string> BrowserExcludedArguments
+        {
+            get
+            {
+                if (excludedArguments == null)
+                {
+                    excludedArguments = SettingsFile.GetValueListOrEmpty<string>($"{DriverSettingsPath}.{nameof(excludedArguments)}");
+
+                    if (excludedArguments.Any())
+                    {
+                        AqualityServices.LocalizedLogger.Debug("loc.browser.excludedArguments", args: string.Join(" ", excludedArguments));
```

Comment:
have you added localization values "loc.browser.excludedArguments"? Can't see them in this PR

Reply:
done